<?php $pageTitle = "Publications" ?>
<?php $pageDescription = "Natural Language Processing Tool Documentation. Clamp Current Users." ?>
<?php include_once dirname(__FILE__) . '/views/header.php'; ?>

<div class="container page-container">
    <br/>
    <div class="section-header">
        <h1>Reference</h1>
    </div> 
     <div>
       <p> Note: Please cite the publication below in study manuscripts using CLAMP. Please inform us of your publication for our reference.
       <br></p>   
       <div class="just-padding">

            <div class="list-group">
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Soysal, E., Wang, J., Jiang, M., Wu, Y., Pakhomov, S., Liu, H., & Xu, H. (2017). CLAMP–a toolkit for efficiently building customized clinical natural language processing pipelines. Journal of the American Medical Informatics Association, 25(3), 331-336.
                    <a href="http://dx.doi.org/10.1093/jamia/ocx132" target="_blank"  style="text-decoration: underline;float: right">Full text link</a>
                </span>
                </div>    
    </div>
    <br/>
    
    <div class="section-header">
        <h1>Awards & Publications</h1>
    </div>

    <div>
        <p>
            CLAMP is built on award-winning methods. Our team has participated in a number of clinical NLP challenges and top ranked for various tasks (see below). Now we have integrated such proven methods into CLAMP and make them available to the community.
        </p>        
    </div>
 
    <div class="just-padding">
        <h3 style="margin-bottom: 20px">Challenges</h3>
        <div class="list-group list-group-root well">
            <p class="list-group-item list-header" >NLP Task: Named entity recognition </p>
            <div class="list-group">
           	    <a class="list-group-item list-sub-header"> 2018 n2c2 Shared Task Track 2 drug, modifiers,  Adverse Drug Events (ADE) <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #2</span></a>
                <a href="#" class="list-group-item list-sub-header"> 2017 TAC-ADR, AdverseReactions and modifier concepts <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #1</span></a>
           	    <a href="#" class="list-group-item list-sub-header"> 2013 SHARe/CLEF abbreviation <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #1</span></a>
           	    <a href="#" class="list-group-item list-sub-header"> 2010 i2b2 problem, treatment, test <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #2</span></a>
           	    <a href="#" class="list-group-item list-sub-header">2009 i2b2, medication <span class="badge badge-pill badge-success"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #2</span></a>
           	    
            </div>

            <p class="list-group-item list-header">NLP Task: Relation extraction</p>
            <div class="list-group">
                <a href="#" class="list-group-item list-sub-header"> 2018 n2c2 Shared Task Track 2 drug-modifer, drug-ADE <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i>#1</span></a>
                <a href="#" class="list-group-item list-sub-header"> 2017 TAC-ADR, AdverseReaction-modifier <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i>#1</span></a>
                <a href="#" class="list-group-item list-sub-header"> 2016 SemEvel, Temporal <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i>#1</span></a>
                <a href="#" class="list-group-item list-sub-header"> 2015 BioCREATIVE Chemical-induced disease <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #1</span></a>
                <a href="#" class="list-group-item list-sub-header"> 2015 SemEval Disease-modifier <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i>#1</span></a>
                <a href="#" class="list-group-item list-sub-header"> 2012 i2b2 Temporal <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #1</span></a>
                
            </div>

            <p class="list-group-item list-header">NLP Task: Encoding</p>
            <div class="list-group">
            
            		<a href="#" class="list-group-item list-sub-header"> 2017 TAC-ADR, AdverseReaction, MedDRA encoding <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i>#1</span></a>
                <a href="#" class="list-group-item list-sub-header"> 2014 SemEval, disorder, UMLS encoding <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i>#1</span></a>
                
            </div>

            <p class="list-group-item list-header">NLP Task: De-identification</p>
            <div class="list-group">
                <a href="#" class="list-group-item list-sub-header">2016 CEGS N-GRID, de-identification <span class="badge"><i class="fa fa-trophy fa-1x" aria-hidden="true" style="margin-right: 5px"></i> #2</span></a>
            </div>
        </div>
    </div>


    <!--  <div class="just-padding">
        <h3 style="margin-bottom: 20px">Relevant Publications</h3>
        <div class="list-group list-group-root well">
        <p class="list-group-item list-header" >2017 </p>   
            <div class="list-group">
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Ergin Soysal, Jingqi Wang, Min Jiang, Yonghui Wu, Serguei Pakhomov, Hongfang Liu, Hua Xu.  CLAMP – a toolkit for efficiently building customized clinical natural language processing pipelines. JAMIA, Doi: 10.1093/jamia/
                    <a href="http://dx.doi.org/10.1093/jamia/ocx132" target="_blank"  style="text-decoration: underline;float: right">Full text link</a>
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Jun Xu*, Hee-Jin Lee*, Zongcheng Ji*, Jingqi Wang, Qiang Wei, and Hua Xu. UTH_CCB System for Adverse Drug Reaction Extraction from Drug Labels at TAC-ADR 2017. Proceedings of TAC, 2017. (* denotes equal contribution)
                    <a target="_blank"  style="text-decoration: underline;float: right">Full text link</a>
                </span>
                </div>
                </div>
        <div class="list-group list-group-root well">
            <p class="list-group-item list-header" >2016 </p>
            <div class="list-group">
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Lee H, Zhang Y, Xu J, Moon S, Wang J, Wu Y, and Xu H, UTHealth at SemEval-2016 Task 12: an End-to-End System for Temporal Information Extraction from Clinical Notes. Proceedings of the 10th International Workshop on Semantic Evaluation (SemEval-2016), 1292-1297, San Diego, California, June, 2016.
                    <a href="https://www.aclweb.org/anthology/S/S16/S16-1201.pdf" target="_blank"  style="text-decoration: underline;float: right">Full text link</a>
                </span>

                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span> Xu J, Wu Y, Zhang Y, Wang J, Lee H-J, Xu H. CD-REST: a system for extracting chemical-induced disease relation in literature. Database: The Journal of Biological Databases and Curation. 2016;2016:baw036. doi:10.1093/database/baw036.
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27016700" target="_blank"  style="text-decoration: underline;float: right">Full text link</a>
                </span>

                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span> Zhang Y, Xu J, Chen H, Wang J, Wu Y, Manu P, Xu H. Chemical named entity recognition in patents by domain knowledge and unsupervised feature learning.Database (Oxford). 2016. doi: 10.1093/database/baw049.
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27087307" target="_blank"  style="text-decoration: underline;float: right">Full text link</a>
                </span>




                </div>
            <p class="list-group-item list-header" >2015 </p>
            <div class="list-group">
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Xu J, Zhang Y, Wang J, Wu Y, Jiang M, Soysal E, and Xu H. UTH-CCB: The Participation of the SemEval 2015 Challenge – Task 14. Proceedings of the 9th International Workshop on Semantic Evaluation (SemEval 2015). Denver, Colorado. 2015:311-314.
                    <a href="https://pdfs.semanticscholar.org/2a6f/5f0da0b833b135731245e755f71bc12645cf.pdf" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Xu J, Wu Y, Zhang Y, Wang J, Liu R, Wei Q, and Xu H. UTH-CCB@BioCreative V CDR Task: Identifying Chemical-induced Disease Relations in Biomedical Text. Proceedings of the Fifth BioCreative Challenge Evaluation Workshop, 2015:254-259.
                    <a href="http://www.biocreative.org/media/store/files/2015/BCV2015_paper_38n.pdf" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Zhang Y, Xu J, Wang J, Wu Y, Parkasam M and Xu H. UTH-CCB@BioCreative V Track 2: Recognizing Chemical Entities in Patents. Proceedings of the Fifth BioCreative Challenge Evaluation Workshop, 2015:147-148
                    <a href="http://www.biocreative.org/media/store/files/2015/BCV2015_paper_62.pdf" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
                </span>


            </div>
            <p class="list-group-item list-header" >2014 </p>
            <div class="list-group">
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Zhang Y, Wang J, Tang B, Wu Y, Jiang M, Chen Y, Xu H. UTH_CCB: A Report for SemEval 2014 – Task 7 Analysis of Clinical Text. SemEval 2014, 802–806, Dublin, Ireland, August 23-24, 2014.
                    <a href="http://www.aclweb.org/anthology/S14-2142" target="_blank"  style="text-decoration: underline;float: right">Full text link</a>
                </span>
                </div>
            <p class="list-group-item list-header" >2013 </p>
            <div class="list-group">
               <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file" ></span>Tang B, Wu Y, Jiang M, and Xu H. A Machine Learning based System for Disorder Concept Extraction. 2013. Proceedings of CLEF 2013 Evaluation Labs and Workshop.
                 <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3168315/" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Wu Y, Tang B, Jiang M, Moon S, Denny JC, Xu H. Clinical Acronym/Abbreviation Normalization using a Hybrid Approach. 2013. Proceedings of CLEF 2013 Evaluation Labs and Workshop.
                    <a href="http://ceur-ws.org/Vol-1179/CLEF2013wn-CLEFeHealth-WuEt2013.pdf"  target="_blank" style="text-decoration: underline;float: right">Full text link</a>
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Tang B, Wu Y, Jiang M, Chen Y, Denny JC, Xu H. A hybrid system for temporal information extraction from clinical text. J Am Med Inform Assoc. 2013.
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/23571849" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
                </span>
                </div>

            <p class="list-group-item list-header" >2011 </p>
            <div class="list-group">
             <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Jiang M Chen Y Liu Met al. A study of machine-learning-based approaches to extract clinical entities and their assertions from discharge summaries. J Am Med Inform Assoc  2011;18:601–6.
                <a href="https://www.ncbi.nlm.nih.gov/pubmed/21508414" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
             </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Jiang M Chen Y Liu Met al. A study of machine-learning-based approaches to extract clinical entities and their assertions from discharge summaries. J Am Med Inform Assoc  2011;18:601–6.
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/21508414" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
                </span>
            </div>

            <p class="list-group-item list-header" >2010 </p>
            <div class="list-group">
            <span  class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Doan S Bastarache L Klimkowski Set al. Integrating existing natural language processing tools for medication extraction from discharge summaries. J Am Med Inform Assoc  2010;17:528–31.
                <a href="https://www.ncbi.nlm.nih.gov/pubmed/20819857" target="_blank" style="text-decoration: underline;float: right">Full text link</a>
            </span>
                </div>
            </div>
        </div>
        -->
        
    <div class="section-header">
        <h1>Publications citing CLAMP</h1>
    		</div> 
    		<div class="just-padding">
    			<div class="list-group">
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Du, J., Zhang, Y., Luo, J., Jia, Y., Wei, Q., Tao, C., & Xu, H. (2018). Extracting psychiatric stressors for suicide from social media using deep learning. BMC medical informatics and decision making, 18(2), 43. 
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Malty, A. M., Jain, S. K., Yang, P. C., Harvey, K., & Warner, J. L. (2018). Computerized approach to creating a systematic ontology of hematology/oncology regimens. JCO clinical cancer informatics, 2, 1-11.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Chen, Q., Du, J., Kim, S., Wilbur, W. J., & Lu, Z. (2018). Combining rich features and deep learning for finding similar sentences in electronic medical records. Proceedings of the BioCreative/OHNLP Challenge.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Li, Y., Yao, L., Mao, C., Srivastava, A., Jiang, X., & Luo, Y. (2018, December). Early prediction of acute kidney injury in critical care setting using clinical notes. In 2018 IEEE International Conference on Bioinformatics and Biomedicine (BIBM) (pp. 683-686). IEEE.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Lee, H. J., Zhang, Y., Jiang, M., Xu, J., Tao, C., & Xu, H. (2018). Identifying direct temporal relations between time and events from clinical notes. BMC medical informatics and decision making, 18(2), 49.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Boag, W., Sergeeva, E., Kulshreshtha, S., Szolovits, P., Rumshisky, A., & Naumann, T. (2018). CliNER 2.0: Accessible and Accurate Clinical Concept Extraction. arXiv preprint arXiv:1803.02245.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Sushil, M., Šuster, S., Luyckx, K., & Daelemans, W. (2018). Patient representation learning and interpretable evaluation using clinical notes. Journal of biomedical informatics, 84, 103-113.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Caufield, J. H., Liem, D. A., Garlid, A. O., Zhou, Y., Watson, K., Bui, A. A., ... & Ping, P. (2018). A Metadata Extraction Approach for Clinical Case Reports to Enable Advanced Understanding of Biomedical Concepts. Journal of visualized experiments: JoVE, (139).
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Van Le, D., Montgomery, J., Kirkby, K. C., & Scanlan, J. (2018). Risk prediction using natural language processing of electronic mental health records in an inpatient forensic psychiatry setting. Journal of biomedical informatics, 86, 49-58.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Luo, Y., & Szolovits, P. (2018, December). Implementing a Portable Clinical NLP System with a Common Data Model--a Lisp Perspective. In 2018 IEEE International Conference on Bioinformatics and Biomedicine (BIBM) (pp. 461-466). IEEE.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Preum, S., Shu, S., Hotaki, M., Williams, R., Stankovic, J., & Alemzadeh, H. CognitiveEMS: A Cognitive Assistant System for Emergency Medical Services.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Yang, X., Bian, J., Gong, Y., Hogan, W. R., & Wu, Y. (2019). MADEx: A System for Detecting Medications, Adverse Drug Events, and Their Relations from Clinical Notes. Drug safety, 1-11.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Caufield, J. H., Zhou, Y., Garlid, A. O., Setty, S. P., Liem, D. A., Cao, Q., ... & Zhang, L. (2018). A reference set of curated biomedical data and metadata from clinical case reports. Scientific data, 5, 180258.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Luo, Y. F., Sun, W., & Rumshisky, A. (2018, June). A Hybrid Method for Normalization of Medical Concepts in Clinical Narrative. In 2018 IEEE International Conference on Healthcare Informatics (ICHI) (pp. 392-393). IEEE.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Chen, X., Gururaj, A. E., Ozyurt, B., Liu, R., Soysal, E., Cohen, T., ... & Rogith, D. (2018). DataMed–an open source discovery index for finding biomedical datasets. Journal of the American Medical Informatics Association, 25(3), 300-308.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Van Le, D., Montgomery, J., Kirkby, K. C., & Scanlan, J. (2018). Risk prediction using natural language processing of electronic mental health records in an inpatient forensic psychiatry setting. Journal of biomedical informatics, 86, 49-58.
                </span>       
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Balyan, R., Crossley, S. A., Brown III, W., Karter, A. J., McNamara, D. S., Liu, J. Y., ... & Schillinger, D. (2019). Using natural language processing and machine learning to classify health literacy from secure messages: The ECLIPPSE study. PloS one, 14(2), e0212488.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Zhang, S., Dev, S., Voyles, J., & Rao, A. S. (2018, December). Attention-Based Multi-Task Learning in Pharmacovigilance. In 2018 IEEE International Conference on Bioinformatics and Biomedicine (BIBM) (pp. 2324-22328). IEEE.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Shi, J., Zheng, M., Yao, L., & Ge, Y. (2018). Developing a healthcare dataset information resource (DIR) based on Semantic Web. BMC medical genomics, 11(5), 102.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Ngwenya, M. (2018). Health systems data interoperability and implementation (Doctoral dissertation).
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Naumann, T. T. J. (2018). Leveraging text representations for clinical predictive tasks (Doctoral dissertation, Massachusetts Institute of Technology).
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Šuster, S., & Daelemans, W. (2018). Revisiting neural relation classification in clinical notes with external information. In Workshop on Health Text Mining and Information Analysis (LOUHI), workshop at EMNLP (pp. 22-28).
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Whalen, P. E., Muralidharan, A. T., Kiddy, J. R., & Duncan, W. D. Enhancing Semantic Analysis of Pathology Reports.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Sastry, K. J., & Ketaraju, V. D. S. A Document Ranking Approach based on Weighted-Gene/Protein in Large Biomedical Documents using MapReduce Framework.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Xu, J., Lee, H. J., Ji, Z., Wang, J., Wei, Q., & Xu, H. (2017). UTH_CCB System for Adverse Drug Reaction Extraction from Drug Labels at TAC-ADR 2017. In TAC.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Newman-Griffis, D., & Zirikly, A. (2018). Embedding Transfer for Low-Resource Medical Named Entity Recognition: A Case Study on Patient Mobility. arXiv preprint arXiv:1806.02814.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Luo, Y. F., Sun, W., & Rumshisky, A. (2019). MCN: A Comprehensive Corpus for Medical Concept Normalization. Journal of Biomedical Informatics, 103132.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Tang, L., Xu, J., Hu, X., Wei, Q., & Xu, H. (2018). Building a Biomedical Chinese-English Parallel Corpus from MEDLINE. MultilingualBIO: Multilingual Biomedical Text Processing, 8.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>GUAN, M. (2018). INCORPORATING EMR AND GENOMIC DATA USING NLP AND MACHINE LEARNING TO REFINE CANCER TREATMENT (Doctoral dissertation, WAKE FOREST UNIVERSITY).
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Redman, J. S., Natarajan, Y., Hou, J. K., Wang, J., Hanif, M., Feng, H., ... & Kanwal, F. (2017). Accurate identification of fatty liver disease in data warehouse utilizing natural language processing. Digestive diseases and sciences, 62(10), 2713-2718.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Rawal, S., Prakash, A., Adhya, S., Kulkarni, S., Anwar, S., Baral, C., & Devarakonda, M. (2019). Developing and Using Special-Purpose Lexicons for Cohort Selection from Clinical Notes. arXiv preprint arXiv:1902.09674.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Karrar, A., Kim, S., Mohammad, M., Addanki, S., Abdelatif, D., Tan, D., ... & Jeffers, T. (2017). Lower Levels of Hepatokines Angiopoietin-Like 4 and RBP-4 are Independently Associated with Non-Alcoholic Steatohepatitis (NASH). Gastroenterology, 152(5), S1115.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Shin, S., Hwang, S. G., Park, H., Lee, J. H., Kwon, S. W., Rim, K. S., & Kim, G. (2017). Diagnostic Performance of Controlled Attenuation Parameter for Hepatic Steatosis in Patient with Chronic Liver Disease, Retrospectively. Gastroenterology, 152(5), S1115.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Névéol, A., & Zweigenbaum, P. (2018). Expanding the Diversity of Texts and Applications: Findings from the Section on Clinical Natural Language Processing of the International Medical Informatics Association Yearbook. Yearbook of medical informatics, 27(01), 193-198.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Dalianis, H. (2018). Computational Methods for Text Analysis and Text Classification. In Clinical Text Mining (pp. 83-96). Springer, Cham.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Zhang, Y., Li, H. J., Wang, J., Cohen, T., Roberts, K., & Xu, H. (2018). Adapting Word Embeddings from Multiple Domains to Symptom Recognition from Psychiatric Notes. AMIA Summits on Translational Science Proceedings, 2017, 281.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Dakhoul, L., Cummings, O. W., Weber, R., & Vuppalanchi, R. (2017). Use of Vibration-Controlled Transient Elastography (VCTE) with Simultaneous Liver Stiffness Measurement (LSM) and Controlled Attenuation Parameter (CAP) in Patients Referred for Suspected Non-Alcoholic Fatty Liver Disease (NAFLD): A Prospective Study. Gastroenterology, 152(5), S1115-S1116.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Fleet, S. E., LeDuc, C. A., & Leibel, R. L. (2017). Elucidating the Role of ILDR2 in the Unfolded Protein Response and its Effect on Nonalcoholic Steatohepatitis. Gastroenterology, 152(5), S1114-S1115.
                </span>
                 <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Ngwenya, M., & Bankole, F. (2019, January). Mining and representing unstructured nicotine use data in a structured format for secondary use. In Proceedings of the 52nd Hawaii International Conference on System Sciences.
                </span>
                <span class="list-group-item" style="padding-bottom: 25px"> <span class="glyphicon glyphicon-file"></span>Zhang, O. R., Zhang, Y., Xu, J., Roberts, K., Zhang, X. Y., & Xu, H. (2017, June). Interweaving Domain Knowledge and unsupervised learning for psychiatric stressor extraction from clinical notes. In International Conference on Industrial, Engineering and Other Applications of Applied Intelligent Systems (pp. 396-406). Springer, Cham.
                </span>
                
                </div>
    	</div>
</div>


</div>

<?php include_once dirname(__FILE__) . '/views/footer.php'; ?>